Tests broken on v2.80.2

Hello,

Thank you for the projet, I don't use it every day but it's awesome.

The tests are broken on the latest release (v2.20.8) and on master.
Here are the errors:

```
$ go test ./...
minify/minify.go:29:26: cannot use &aspMinifier (value of type **"github.com/tdewolff/minify/v2/html".Minifier) as "github.com/tdewolff/minify/v2".Minifier value in argument to Default.Add: **"github.com/tdewolff/minify/v2/html".Minifier does not implement "github.com/tdewolff/minify/v2".Minifier (missing method Minify)
minify/minify.go:30:37: cannot use &aspMinifier (value of type **"github.com/tdewolff/minify/v2/html".Minifier) as "github.com/tdewolff/minify/v2".Minifier value in argument to Default.Add: **"github.com/tdewolff/minify/v2/html".Minifier does not implement "github.com/tdewolff/minify/v2".Minifier (missing method Minify)
minify/minify.go:34:41: cannot use &phpMinifier (value of type **"github.com/tdewolff/minify/v2/html".Minifier) as "github.com/tdewolff/minify/v2".Minifier value in argument to Default.Add: **"github.com/tdewolff/minify/v2/html".Minifier does not implement "github.com/tdewolff/minify/v2".Minifier (missing method Minify)
minify/minify.go:38:36: cannot use &tmplMinifier (value of type **"github.com/tdewolff/minify/v2/html".Minifier) as "github.com/tdewolff/minify/v2".Minifier value in argument to Default.Add: **"github.com/tdewolff/minify/v2/html".Minifier does not implement "github.com/tdewolff/minify/v2".Minifier (missing method Minify)
minify/minify.go:39:42: cannot use &tmplMinifier (value of type **"github.com/tdewolff/minify/v2/html".Minifier) as "github.com/tdewolff/minify/v2".Minifier value in argument to Default.Add: **"github.com/tdewolff/minify/v2/html".Minifier does not implement "github.com/tdewolff/minify/v2".Minifier (missing method Minify)
minify/minify.go:40:44: cannot use &tmplMinifier (value of type **"github.com/tdewolff/minify/v2/html".Minifier) as "github.com/tdewolff/minify/v2".Minifier value in argument to Default.Add: **"github.com/tdewolff/minify/v2/html".Minifier does not implement "github.com/tdewolff/minify/v2".Minifier (missing method Minify)
# github.com/tdewolff/minify/v2/minify [github.com/tdewolff/minify/v2/minify.test]
```

Have a good day. 👍

--- a/minify/minify.go
+++ b/minify/minify.go
@@ -24,16 +24,16 @@ func init() {
 	Default.AddFuncRegexp(regexp.MustCompile("[/+]json$"), json.Minify)
 	Default.AddFuncRegexp(regexp.MustCompile("[/+]xml$"), xml.Minify)
 
-	aspMinifier := &html.Minifier{}
+	aspMinifier := html.Minifier{}
 	aspMinifier.TemplateDelims = [2]string{"<%", "%>"}
 	Default.Add("text/asp", &aspMinifier)
 	Default.Add("text/x-ejs-template", &aspMinifier)
 
-	phpMinifier := &html.Minifier{}
+	phpMinifier := html.Minifier{}
 	phpMinifier.TemplateDelims = [2]string{"<?", "?>"} // also handles <?php
 	Default.Add("application/x-httpd-php", &phpMinifier)
 
-	tmplMinifier := &html.Minifier{}
+	tmplMinifier := html.Minifier{}
 	tmplMinifier.TemplateDelims = [2]string{"{{", "}}"}
 	Default.Add("text/x-go-template", &tmplMinifier)
 	Default.Add("text/x-mustache-template", &tmplMinifier)

